piwigo.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Piwigo",
+    "plugin_schema": "2",
+    "release": "11.4-RELEASE",
```

Comment:
```suggestion
    "release": "12.2-RELEASE",
```

Reply:
I will test this now if this works. Give me 10min ;)

Reply:
Got it to work without problems, Release 12.2 is working, confirmed ;)

Reply:
`code`